web3/types.py

```diff
@@ -368,6 +368,7 @@ class FeeHistory(TypedDict):
 TxReceipt = TypedDict(
     "TxReceipt",
     {
+        "accessList": AccessList,
```

Comment:
I don't see `accessList` coming through on transaction receipts. Is there a reason we added it here?

Reply:
I wasnt sure what type it belonged to. Seemed like the typing in my `eth_module` test was messed up when I tried integrating the `create_access_list` function. `TxData` doesnt have a `gasUsed` param, which seemed appropriate since the `gasUsed` should be in the response of the `create_access_list` function.

Any recommendations on the typings?

Reply:
Oh yeah I missed this entirely. We should not return a `TxReceipt` as that is not the return value of this method. `TxReceipt` is meant for `getTransactionReceipt`. It looks like the `result` object here returns only `accessList` and `gasUsed`, therefore we should define a `TypedDict` named `CreateAccessListResponse` with only these two fields and probably keep `total=True` (which is default so no need to override it) since it looks like that will be returned every time.